android/app/src/main/java/org/fossasia/openevent/fragments/DayScheduleFragment.java

```diff
@@ -39,7 +39,7 @@
 /**
  * Created by Manan Wason on 17/06/16.
  */
-public class DayScheduleFragment extends Fragment {
+public class DayScheduleFragment extends Fragment implements SwipeRefreshLayout.OnRefreshListener {
```

Comment:
you mean remove the global listener and keep the the anonymous listener (which i removed) it was because so I could reuse the ` onrefresh() ` method in case of retry action.